src/main/java/seedu/address/model/person/LicencePlate.java

```diff
@@ -31,14 +31,14 @@ public class LicencePlate {
     public LicencePlate(String licencePlate) {
         requireNonNull(licencePlate);
         checkArgument(isValidLicencePlate(licencePlate), MESSAGE_CONSTRAINTS);
-        value = licencePlate;
+        value = licencePlate.toUpperCase();
```

Comment:
Good way to ensure that the licence plate is always stored in upper case for the letters!